frontend/src/layouts/header-layout.tsx

```diff
@@ -53,30 +53,30 @@ function header() {
   return (
     <div className="relative -mt-2 h-[460px] bg-[url('./assets/wanderlustbg.webp')] bg-cover bg-fixed bg-center">
       <div className="absolute inset-0 bg-black opacity-50"></div>
-      <div className="absolute inset-0 flex flex-col px-4 py-8 text-slate-50 sm:px-16">
+      <div className="absolute inset-0 flex flex-col px-4 py-8 text-slate-50 sm:px-8 lg:px-16">
         <div className="flex w-full justify-between">
```

Comment:
please remove lg: and it should ideally be px-8 and sm:px-16, figure out why, its interesting

Reply:
![Screenshot_20240517-125027.png](https://github.com/krishnaacharyaa/wanderlust/assets/114509045/0a4ff765-a691-4a0c-ba23-93feac78ba1d)

I made changes to make it aligned with the blog feed which has css on line no. 27.

Reply:
> please remove lg: and it should ideally be px-8 and sm:px-16, figure out why, its interesting

If i remove lg and keep the changes to px-8 and sm:px-16 alignment doesn't look good.

I'll remove it if it is not needed.

Reply:
> please remove lg: and it should ideally be px-8 and sm:px-16, figure out why, its interesting

Sure.